Makefile

```diff
@@ -0,0 +1,13 @@
+help:
+	@echo "start - starts all services"
+	@echo "stop  - stops all services and removes anonymous volumes"
+	@echo "shell - runs an instance of the 'app' service and presents a bash prompt"
+
+start:
+	docker-compose up
+
+stop:
```

Comment:
A bit asymmetric wrt `docker-compose stop`, which stops the containers but doesn't destroy the state (the volumes)

Reply:
The make target is stop. The command is `docker-compose down -v`.

Reply:
Yes, `stop` doesn't just stop the containers with this command but also deletes data; it's local so it should not be important state...

Reply:
Would you like me to remove `-v`?

Reply:
Between the two options of 1) named `stop` but it deletes data and 2) leftover data from previous runs, I'd go for 1) for a development environment (leaving as it is).